test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -77,7 +81,9 @@ build_branch()
     else
         config+="--enable-snappy "
         config+="--disable-standalone-build "
-        (mkdir build && cd build && sh ../build_posix/reconf &&
+
+        # Check if the build directory already exists, if so recompile the code without remaking the directory
+        (mkdir -p build && cd build && sh ../build_posix/reconf &&
```

Comment:
Though I don't think the Evergreen task will need the change (as it's always creating a directory & building from scratch), it could be helpful if users would like to run the script manually a few times. Do you mind port the same change to line 79 above?

Reply:
Sure thing!